lib/ansible/modules/storage/hpe/hpe3par_cpg.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/python
+
+# (C) Copyright 2018 Hewlett Packard Enterprise Development LP
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.  Alternatively, at your
+# choice, you may also redistribute it and/or modify it under the terms
+# of the Apache License, version 2.0, available at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <https://www.gnu.org/licenses/>
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+author: "Hewlett Packard Enterprise (ecostor@groups.ext.hpe.com)"
+description:
+  - "Create and delete CPG on HPE 3PAR."
```

Comment:
Not done but not a mandatory change. Just in case, here is an [example](https://github.com/ansible/ansible/blob/f89d873698c0d22c2f1692938854db0fe3d0acb0/lib/ansible/modules/cloud/amazon/aws_kms_facts.py#L170).